src/lib-components/Page/index.tsx

```diff
@@ -88,9 +101,9 @@ export function Page({
       <MainDiv
         style={{
           minHeight: `calc(100vh - ${diff}px)`,
-          height: `calc(100vh - ${diff}px)`,
-          alignItems: centralized ? 'center' : 'normal',
-          justifyContent: centralized ? 'center' : 'normal',
+          height: `max(calc(100vh - ${diff}px), ${childrenHeight})`,
+          alignItems: centralized || alignItems ? 'center' : 'normal',
+          justifyContent: centralized || justifyContent ? 'center' : 'normal',
```

Comment:
Would be better if alignItems/justifyContent are defined to use their exact value, and not supose they will be used just to centralize